website/templates/project/addon/widget.mako

```diff
@@ -9,14 +9,17 @@
 
                 </div>
             </div>
+    % else:
+        <div>
+    % endif
 
     % if complete:
 
         <div class="panel-body">
             ${self.body()}
         </div>
 
-    % else:
+    % elif not complete and 'admin' in user['permissions']:
```

Comment:
…users with write permission can have the ability to configure addons, I believe. So play around with what you can do as a user with different permissions as far as configuring addons and then revisit this logic.